contracts/token/RefundableToken.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^0.4.8;
+
+
+import './StandardToken.sol';
+import '../ownership/Ownable.sol';
+import '../SafeMath.sol';
+
+
+/**
+ * @title Refundable token
+ * 
+ * @dev Simple ERC20 Token example, with refundable token creation.
+ * @dev Issue: https://github.com/OpenZeppelin/zeppelin-solidity/issues/217
+ *
+ * @notice Your main contract should inherit form this contract and implement the canRefund() method.
+ * @notice Warning -> If you do not implement canRefund() this will be an abstract contract.
+*/
+
+
+contract RefundableToken is StandardToken, Ownable {
+  event Refund(address indexed to, uint amount);
```

Comment:
This is a problem of the Ethereum ecosystem in general but it's not clear here what the `amount` member represents. Is it a token amount or a wei amount of ether?

I can see from the implementation that it represents ether, but note that the `_amount` argument of the `refund` function is insteand an amount of token!

What are your thoughts on this?